media_player/alexa.py

```diff
@@ -1085,3 +1110,27 @@ def get_devices(url, session):
             _LOGGER.error("An error occured accessing the API: {}".format(
                 message))
             return None
+
+    @staticmethod
```

Comment:
This does not have to be a static method. It is a call within the instantiated object of the Alexa Device class. It should be converted to a static method and the calls within the refresh method changed to use the instantiated alexa_api class instead of a static class member.